Content.Server/Chemistry/ReagentEffects/HealthChange.cs

```diff
@@ -15,17 +16,25 @@ namespace Content.Server.Chemistry.ReagentEffects
     /// </summary>
     public class HealthChange : ReagentEffect
     {
+
         /// <summary>
         /// How much damage is changed when 1u of the reagent is metabolized.
         /// </summary>
         [DataField("healthChange")]
         public float AmountToChange { get; set; } = 1.0f;
 
         /// <summary>
-        /// Class of damage changed, Brute, Burn, Toxin, Airloss.
+        /// Damage group to change.
         /// </summary>
-        [DataField("damageClass", true)]
-        public string damageType { get; set; } = default!;
+        // TODO PROTOTYPE Replace this datafield variable with prototype references, once they are supported.
+        // Also requires replacing DamageGroup() calls with something like _damageGroup
+        [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
+        [DataField("damageGroup", required: true)]
+        private readonly string _damageGroupID = default!;
+        private DamageGroupPrototype DamageGroup() {
+            IoCManager.InjectDependencies(this);
+            return _prototypeManager.Index<DamageGroupPrototype>(_damageGroupID);
+        }
```

Comment:
uhh what is this, why does it inject manually, just resolve prototypeManager in Metabolize() and don't do this

Reply:
Yeah, thats probably better. Not sure what exactly IoCManager.InjectDependencies does, it was just mentioned by someone in howdoicode, though they didn't have full context.